src/gpu/nodeKernels.cu

```diff
@@ -228,20 +279,19 @@ void gpuFilter(cudaStream_t stream, size_t count, const Field<RAY_IDX_U32>::type
 	run(kFilter, stream, count, indices, dst, src, fieldSize);
 }
 
-void gpuFilterGroundPoints(cudaStream_t stream, size_t pointCount, const Vec3f sensor_up_vector,
-                           float ground_angle_threshold, const Field<XYZ_VEC3_F32>::type* inPoints,
-                           const Field<NORMAL_VEC3_F32>::type* inNormalsPtr, Field<IS_GROUND_I32>::type* outNonGround,
-                           Mat3x4f lidarTransform)
+void gpuFilterGroundPoints(cudaStream_t stream, size_t pointCount, const Vec3f sensor_up_vector, float ground_angle_threshold,
+                           const Field<XYZ_VEC3_F32>::type* inPoints, const Field<NORMAL_VEC3_F32>::type* inNormalsPtr,
+                           Field<IS_GROUND_I32>::type* outNonGround, Mat3x4f lidarTransform)
 {
 	run(kFilterGroundPoints, stream, pointCount, sensor_up_vector, ground_angle_threshold, inPoints, inNormalsPtr, outNonGround,
 	    lidarTransform);
 }
 
 void gpuRadarComputeEnergy(cudaStream_t stream, size_t count, float rayAzimuthStepRad, float rayElevationStepRad, float freq,
-                           const Field<RAY_POSE_MAT3x4_F32>::type* rayPose, const Field<DISTANCE_F32>::type* hitDist,
-                           const Field<NORMAL_VEC3_F32>::type* hitNorm, const Field<XYZ_VEC3_F32>::type* hitPos,
-                           Vector<3, thrust::complex<float>>* outBUBRFactor)
+                           Mat3x4f lookAtOriginTransform, const Field<RAY_POSE_MAT3x4_F32>::type* rayPose,
```

Comment:
const& or pointer on this would make sense?

Reply:
const& - ref to host memory
pointer - copy needed anyway
I leave it as it is